### PR TITLE
Update About -> Links page

### PR DESCRIPTION
**Pull request by connersaeli**
## Description

Updated this page for ACCESS transition. Images and their corresponding links were changed.

image
image

## Motivation and Context

https://app.asana.com/0/1202679277743034/1202852322868255

## Tests performed

Tested on my development port.

## Checklist:

- [ ] The pull request description is suitable for a Changelog entry
- [ ] The milestone is set correctly on the pull request
- [ ] The appropriate labels have been added to the pull request

### CODE REVIEW

File: html/about/links.html [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+  <td div align=center> <a href="https://access-ci.org" target="_blank" rel="noopener noreferrer"><img src="/about/images/access_logo.png"></a> </td>
 </tr>
 <tr>
   <td div align=center> <a href="http://www.buffalo.edu/ccr.html" target="_blank" rel="noopener noreferrer"><img src="/about/images/Ccr_ub_logo.jpg" width="500"></a>
   <td div align=center> <a href="http://www.nsf.gov" target="_blank" rel="noopener noreferrer"><img src="/about/images/nsf_logo.png"></a>
 </tr>
 <tr>
-  <td div align=center> <a href="https://www.tacc.utexas.edu/" target="_blank" rel="noopener noreferrer"><img src="/about/images/TACC_logo.png" width=300"></a>
-  <td div align=center> <a href="http://www.iub.edu/" target="_blank" rel="noopener noreferrer"><img src="/about/images/indianauniversity_logo.jpg" height=200></a>
+  <td div align=center> <a href="https://www.tacc.utexas.edu/" target="_blank" rel="noopener noreferrer"><img src="/about/images/TACC_logo.png" width="300"></a>
```
Comment (jpwhite4) [inline html/about/links.html:12]: how come this line shows as changed in the diff?

File: html/about/links.html [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+  <td div align=center> <a href="http://www.iub.edu/" target="_blank" rel="noopener noreferrer"><img src="/about/images/indianauniversity_logo.jpg" height="200"></a>
+</tr>
+<tr>
+  <td div align=center> <a href="https://www.sdsc.edu/" target="_blank" rel="noopener noreferrer"><img src="/about/images/sdsc_logo.png" height="100" width="300"></a>
```
Comment (jpwhite4) [inline html/about/links.html:16]: This changes the aspect ratio of the logo, which is not a good idea since it has text in it.

Comment (jpwhite4) [reply html/about/links.html:16]: Why not use a logo that has the same shape as the others?

https://www.sdsc.edu/assets/images/logos/SDSClogo-plusname-red.jpg

**Review by jpwhite4 — CHANGES_REQUESTED** [review]

File: html/about/links.html (hunk 1 above)
Comment (connersaeli) [reply html/about/links.html:12]: There was a floating double quote after 'width' for the TACC line that I closed and I just changed the following line to match that formatting.

File: html/about/links.html (hunk 2 above)
Comment (connersaeli) [reply html/about/links.html:16]: Updated with suggested logo:
image

Comment (connersaeli) [reply html/about/links.html]: There is no longer height/width modifier for this update --> original aspect ratio is preserved

File: html/about/links.html (hunk 1 above)
Comment (jpwhite4) [reply html/about/links.html:12]: ah well spotted - I was staring at that line for ages trying to work out what was different

**Comment by connersaeli** [comment]
Resize Tufts logo to match others:
image

**Review by jpwhite4 — DISMISSED** [review]

**Pull request closed without merging**